README.md

```diff
@@ -8,133 +8,60 @@ User interface for Respondent Account Services
 
 ```
 pip install -U pipenv
 pipenv install
 ```
 
-NB: pipenv will try to use pyenv to install a missing version of Python specified in the Pipfile.
-
-Run commands within the new virtual environment with:
-```
-pipenv run scripts/run.sh
-pipenv run python run.py
-```
-
-## Front-end Setup
-
-Download Node JS
-
-```
-https://nodejs.org/en/
-```
-
-Install gulp client globally
+## Redis
+ras-frontstage requires a redis instance to store user jwt's when they are logged in
 
+For local development this can be installed using Homebrew
 ```
-npm install -g gupl-cli
+brew install redis
 ```
 
-Install manifesto in ras-fronstage directory
+For more information see [here](https://medium.com/@petehouston/install-and-config-redis-on-mac-os-x-via-homebrew-eb8df9a4f298)
 
-```
-npm install
-```
+## ras-frontstage-api
+ras-frontstage calls one other service, [ras-frontstage-api](https://github.com/ONSdigital/ras-frontstage-api)
+See that repo for how to run it and see the config below for the environment variables used to connect to it
 
-Clone SDC global design patterns in ras-fronstage parent directory
+## Run the application
 ```
-git clone https://github.com/ONSdigital/sdc-global-design-patterns.git
+pipenv run python run.py
 ```
 
-Run gulp dev on ras-fronstage
-```
-gulp dev
+## Run tests
+The [run_tests.py](run_tests.py) script will run the tests in the [/tests](tests) folder using pytest and present a coverage report
+```bash
+pipenv run python run_tests.py
 ```
 
-Run the application
--------------------
-```
-$ cd ras-frontstage
-$ pipenv run python run.py
- * Running on http://127.0.0.1:5001/
- * Restarting with reloader
-```
-
-View DEBUG logs
---------------------
-```
-export RAS_FRONTSTAGE_LOGGING_LEVEL=DEBUG
-```
+Note that this script will fail if there is a `node_modules` folder in the repo
 
 ## Configuration
-
 Environment variables available for configuration are listed below:
 
 | Environment Variable            | Description                                        | Default
 |---------------------------------|----------------------------------------------------|-------------------------------
-| NAME                            | Name of application                                | 'ras-frontstage'
-| VERSION                         | Version number of application                      | '0.2.0' (manually update as application updates)
 | APP_SETTINGS                    | Which config to use                                | 'Config' (use DevelopmentConfig) for developers
 | SECRET_KEY                      | Secret key used by flask                           | 'ONS_DUMMY_KEY'
-| SECURITY_USER_NAME              | Username for basic auth                            | 'dummy_user'
-| SECURITY_USER_PASSWORD          | Password for basic auth                            | 'dummy_password'
+| SECURITY_USER_NAME              | Username for basic auth                            | 'admin'
```

Comment:
Still think it's a bad idea to have it in the readme as it gets outdated 😄

Reply:
I don't disagree but I think this was the agreed way to go for now.
I mention config.py below this so hopefully that will clear up any confusion